contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1789,20 +1796,20 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     function _resetContext()
         internal
     {
-        transactionContext.ovmL1TXORIGIN = address(0);
-        transactionContext.ovmTIMESTAMP = 0;
-        transactionContext.ovmNUMBER = 0;
-        transactionContext.ovmGASLIMIT = 0;
-        transactionContext.ovmTXGASLIMIT = 0;
+        transactionContext.ovmL1TXORIGIN = MAX_ADDRESS;
+        transactionContext.ovmTIMESTAMP = type(uint256).max;
+        transactionContext.ovmNUMBER = type(uint256).max;
+        transactionContext.ovmGASLIMIT = type(uint256).max;
+        transactionContext.ovmTXGASLIMIT = type(uint256).max;
         transactionContext.ovmL1QUEUEORIGIN = Lib_OVMCodec.QueueOrigin.SEQUENCER_QUEUE;
 
         transactionRecord.ovmGasRefund = 0;
```

Comment:
This is the only value I didn't set to `type(uint256).max`. The reason being that several test cases failed, because it seems that in those cases (mostly static call violations), the default gas refund value is used.

Reply:
If we ever revive this, I'm now of the opinion that something like `0x000000000000000000000000000000000000000000000000000000000000dead` would be better. I don't like being within an 'off-by-one' error of zero.